components/home/modules/ROOT/pages/index.adoc

```diff
@@ -1,64 +1,86 @@
 
+== Why OpenZeppelin?
 
-== Why use OpenZeppelin tools?
+OpenZeppelin tools have been built with *security in mind*. Since the first release of the Contracts library in 2016, our libraries have been downloaded over 1.5 million times to create tokens, protect digital assets, and power decentralized applications, securing more than $4.5 billion in digital assets. We also work with top Ethereum projects through our https://openzeppelin.com/security-audits/[security audits service].
 
-OpenZeppelin has been the bedrock of smart contract development in Ethereum for the last few years. Our libraries have been used by more than 100K developers to create tokens, protect digital assets and power decentralized applications. It has helped developers secure more than $4.5 billion in digital assets.
+== Learn
 
-As the ecosystem matures, we are taking all the lessons we have learned during this journey to help you create professional applications that are ready for consumer adoption.
+Comprehensive guides for every step of your development journey
+
+[horizontal]
+Environment Setup:: Install node, npm, and the basic libraries for web3 development
+Write Smart Contracts:: Write and compile your first Solidity contracts, using the building blocks from the OpenZeppelin Contracts library
+Local Development:: Deploy your contracts to a local blockchain and send your first transactions using the OpenZeppelin CLI
+Test your Contracts:: Unit test your contracts with the OpenZeppelin `test-environment`, combined with assertions from our `test-helpers`
+Upgrade your Contracts:: Change the code of your deployed contracts to add new features or fix bugs, using the OpenZeppelin CLI or the `upgrades` library
+Deploy to Testnet:: Set up a staging environment by deploying your app to a public Ethereum testnet using the OpenZeppelin CLI
+Build a Dapp:: Create a decentralized web app using OpenZeppelin `network-js` and `hot-loader`, or simply unpack a Starter Kit
+Send Gasless Transactions:: Leverage the Gas Station Network so your users can send transactions without having ETH to pay for gas
+Go to Mainnet:: All the boxes you need to check before taking your project to production on the main Ethereum network
+
+== Tools
+
+All the open source tools built and maintained by the OpenZeppelin team
 
 [.card.card-contracts]
 .xref:contracts::index.adoc[Contracts]
 --
-A library of modular, reusable, secure smart contracts, written in Solidity.
+A library of modular, reusable, and secure smart contracts, written in Solidity.
 
 [.card-cta]#xref:contracts::index.adoc[Get started >]#
 --
 
 [.card.card-sdk]
-.xref:sdk::index.adoc[SDK]
+.xref:sdk::index.adoc[CLI]
 --
-A suite of tools to help you develop, test, upgrade and deploy smart contracts.
+A command-line tool to help you deploy, upgrade, and interact with your smart contracts.
```

Comment:
I'd change all of these descriptions into actions, such as 'Deploy, upgrade and interact with your smart contracts from the command line'

Reply:
Hmmm disagree. I wanted to emphasize what every tool is here, as opposed to the actions which were emphasized in the Learn section.

Reply:
Note though that they all follow this format:

>A library/tool for doing X

I'd leave this as-is though, this will come up naturally once/if we add the badges mentioned in the other comment.